docker-compose.yaml

```diff
@@ -0,0 +1,19 @@
+version: '3.1'
```

Comment:
Docker-compose file to simplify local development - it starts MySQL server with connection properties matching those from `persistence.xml` and MySQL viewer which is available on http://localhost:8020/